src/index.html

```diff
@@ -164,5 +166,41 @@ <h2 class="modal__title">New post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button class="modal__close-btn" type="button"></button>
+
+        <h2 class="modal__title">Edit Avatar</h2>
+        <form class="modal__form" id="edit-avatar-modal" name="edit-avatar">
+          <label class="modal__label" for="profile-avatar-input">
+            Avatar
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/xaSFJc.jpg 


```html
            <input
              type="url"
```